mmv1/templates/terraform/examples/target_https_proxy_mtls.tf.erb

```diff
@@ -0,0 +1,93 @@
+# [START cloudloadbalancing_target_https_proxy_mtls]
+data "google_project" "project" {
+  provider = google-beta
+}
+
+resource "google_compute_target_https_proxy" "<%= ctx[:primary_resource_id] %>" {
+  provider          = google-beta
+  name              = "<%= ctx[:vars]['target_https_proxy_name'] %>"
+  url_map           = google_compute_url_map.default.id
+  ssl_certificates  = [google_compute_ssl_certificate.default.id]
+  server_tls_policy = google_network_security_server_tls_policy.default.id
+}
+
+resource "google_certificate_manager_trust_config" "default" {
+  provider    = google-beta
+  name        = "<%= ctx[:vars]["trust_config_name"] %>"
+  description = "sample description for the trust config"
+  location    = "global"
+
+  trust_stores {
+    trust_anchors {
+      pem_certificate = file("test-fixtures/ca_cert.pem")
+    }
+    intermediate_cas {
+      pem_certificate = file("test-fixtures/ca_cert.pem")
+    }
+  }
+
+  labels = {
+    foo = "bar"
+  }
+}
+
+resource "google_network_security_server_tls_policy" "default" {
+  provider               = google-beta
+  name                   = "<%= ctx[:vars]['server_tls_policy_name'] %>"
+  description            = "my description"
+  location               = "global"
+  allow_open             = "false"
+  mtls_policy {
+    client_validation_mode = "ALLOW_INVALID_OR_MISSING_CLIENT_CERT"
+    client_validation_trust_config = "projects/${data.google_project.project.number}/locations/global/trustConfigs/${google_certificate_manager_trust_config.default.name}"
```

Comment:
At first I tried using the `id` of the `google_certificate_manager_trust_config` resource in this parameter. However, when the tests executed the same configuration resulted in a plan that wanted to update the `tls_policy` from `client_validation_trust_config = projects/${data.google_project.project.id}/locations/global/trustConfigs/${google_certificate_manager_trust_config.default.name}` to `client_validation_trust_config = projects/${data.google_project.project.number}/locations/global/trustConfigs/${google_certificate_manager_trust_config.default.name}` (i.e. the trust config reference link used the project number in the API but the project ID in the terraform state). I'm not sure if this is a problem with my example, a bug with the `trust_config` resource, or a bug with the `server_tls_policy` resource.